Makefile

```diff
@@ -32,7 +32,15 @@ govet:
 	go vet ./...
 
 gotest:
-	go test -count=1 -short -v -cover ./...
+	go test -count=1 -short -v -coverprofile cover.out ./...
+
+.PHONY: analyze-cover-profile
+analyze-cover-profile: cover-profile
+	./scripts/analyze-cover-profile
+
+.PHONY: cover-profile
+cover-profile:
```

Comment:
nit: if you name this the actual name of the file that gets generated, you don't need to declare it phony.

Reply:
oh, also this should depend on gotest

Reply:
true, I actually had it that way to begin with. Drawback though is that then it won't regenerate the file when it sees it's already there so re-analyzing changed code would require cleaning up this file. Since all our other targets are "phony" I thought this would be confusing to most people as we don't typically need a "make clean" in between make targets atm.

Reply:
also, `make clean` should cleanup cover.out and coverprofile.out

Sorry for the comment stream, brain is stalling.

Reply:
> oh, also this should depend on gotest

we dont want to rerun gotest when this runs though, the cover.out file is already made